src/api/app/models/event_find_subscriptions.rb

```diff
@@ -31,86 +31,76 @@ def subscriptions
   def expand_toconsider
     new_toconsider = []
     @toconsider.each do |subscription|
-      new_toconsider.concat(expand_one_rule(subscription))
+      new_toconsider.concat(expand_subscription(subscription))
     end
     @toconsider = new_toconsider
   end
 
-  def expand_one_rule(r)
-    if r.receiver_role == :all
```

Comment:
How do you know this is safe to do?

Reply:
Because there was no interface to create `EventSubscription`s with `receiver_role` `:all`, there is no `Event` with this as `receiver_role` and there are no such `EventSubscriptions` on the reference server.